inlong-sort/sort-connectors/jdbc/src/main/java/org/apache/inlong/sort/jdbc/dialect/PostgresDialect.java

```diff
@@ -137,4 +146,71 @@ public List<LogicalTypeRoot> unsupportedTypes() {
                 LogicalTypeRoot.SYMBOL,
                 LogicalTypeRoot.UNRESOLVED);
     }
+
+    @Override
+    public List<String> getAndSetPkNamesFromDb(String tableIdentifier, JdbcOptions jdbcOptions) {
```

Comment:
Why is it called Get and Set? I just see a Get operation in the implementation logic.